test/extended/setup.sh

```diff
@@ -6,12 +6,12 @@
 function os::test::extended::focus {
   if [[ $# -ne 0 ]]; then
     echo "[INFO] Running custom: $@"
```

Comment:
My comment to use `$*` was for this line -- technically it's mixing a string (the literal) with an array (`$@`), so the more correct expression would be:
```sh
$ echo "[INFO] Running custom: $*"
```